bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/Parameters.scala

```diff
@@ -1,16 +1,29 @@
 package edu.gemini.itc.shared
 
-import edu.gemini.spModel.core.{SpectralDistribution, UniformSource, SpatialProfile, Redshift, BrightnessUnit, MagnitudeBand}
-import edu.gemini.spModel.gemini.obscomp.SPSiteQuality.{WaterVapor, CloudCover, SkyBackground, ImageQuality}
+import edu.gemini.shared.util.immutable.ImEither
+import edu.gemini.spModel.core.{BrightnessUnit, MagnitudeBand, Redshift, SpatialProfile, SpectralDistribution, UniformSource}
+import edu.gemini.spModel.gemini.obscomp.SPSiteQuality.{CloudCover, ImageQuality, SkyBackground, WaterVapor}
+
+import scalaz.\/
 
 // ==== Observing conditions
 
 final case class ObservingConditions(
-                      iq: ImageQuality,
-                      cc: CloudCover,
+                      iq: Double \/ ImageQuality,
```

Comment:
Maybe me can just use `Either`? This code is being ported to scala 2.13 without `scalaz`

Reply:
I started with `Either` but then I saw `\/` was used elsewhere in the ITC (e.g., `ConfigExtractor`) so I thought it best to be consistent with that.

Reply:
I guess that's ok, we'll have to port the rest anyway.

At any rate, great not to modify the enum, I started fixing bugs due to that change and realized it would be fragile to have this extra entry

Reply:
This change is geared towards supporting exact iq and cc values on gpp however in `lucuma-core` we have that IQ and CC are limited to a strict set of values:

https://github.com/gemini-hlsw/lucuma-core/blob/master/modules/core/shared/src/main/scala/lucuma/core/enum/ImageQuality.scala#L16
https://github.com/gemini-hlsw/lucuma-core/blob/master/modules/core/shared/src/main/scala/lucuma/core/enum/CloudExtinction.scala#L9

Should we make the `lucuma-core` version a non-enum and thus remove (in the gpp itc) the need for an `Either`?